interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/open-reports.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Open Reports
+author: Mary Sammal
+---
+# Open Reports
+
+You can use one of the following commands to open a report. The invoked **Save** dialog allows you to locate and open previously [saved reports](save-reports.md).
+
+### Use the Ribbon Application Menu
+
+Click **Open...**.
+
+![eurd-win-ribbon-open-reports](../../../images/eurd-win-ribbon-open-reports.png)
+
+
+### Use the Ribbon Application Menu
+
+Click the application button and then **Open...** in the invoked application menu.
```

Comment:
"the application button"?

Reply:
This button is demonstrated on the image that follows this sentence.
![image](https://user-images.githubusercontent.com/4559556/42448945-9d78572a-8387-11e8-8e1d-025dc44d640c.png) (switch to Preview)

Reply:
This is what it is called? The 'application' button?

Reply:
Yes. https://documentation.devexpress.com/WindowsForms/8313/Controls-and-Libraries/Ribbon-Bars-and-Menu/Ribbon/Visual-Elements/Application-Button